packages/server/graphql/types/Team.ts

```diff
@@ -228,7 +228,7 @@ const Team = new GraphQLObjectType<ITeam, GQLContext>({
         const viewerId = getUserId(authToken)
         const allTasks = await dataLoader.get('tasksByTeamId').load(teamId)
         const tasks = allTasks.filter((task) => {
-          if (isTaskPrivate(task.tags) && task.userId !== viewerId) return false
+          if (!task.userId || (isTaskPrivate(task.tags) && task.userId !== viewerId)) return false
```

Comment:
We could use the tasks query here instead of the resolve function. As both could work, I thought I'd keep things as they are and filter from here.